src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/Internal/Topics.cs

```diff
@@ -11,28 +11,53 @@
 using System.Linq;
 using System.Runtime.CompilerServices;
 using Akka.Actor;
+using Akka.Event;
 using Akka.Remote;
 using Akka.Routing;
 
 namespace Akka.Cluster.Tools.PublishSubscribe.Internal
 {
     /// <summary>
-    /// TBD
+    /// A <see cref="DeadLetter"/> published when there are no subscribers
+    /// for a topic that has received a <see cref="Publish"/> event.
+    /// </summary>
+    internal readonly struct NoSubscribersDeadLetter
```

Comment:
Created an envelope `NoSubscribersDeadLetter` in order to help make it clear inside the user's logging system why this `DeadLetter` was published. Created it as a `readonly struct` because it will immediately be captured inside a `DeadLetter` and therefore we can save on an allocation here in the event of a large number of deadletters being published all at once.